authentik/core/sources/flow_manager.py

```diff
@@ -342,7 +341,11 @@ def handle_auth(
                     ),
                 )
             ],
-            **flow_kwargs,
+            **{
+                PLAN_CONTEXT_PENDING_USER: connection.user,
+                PLAN_CONTEXT_PROMPT: delete_none_values(self.user_properties),
+                PLAN_CONTEXT_USER_PATH: self.source.get_user_path(),
```

Comment:
This might override what was set by property mappings.

Reply:
Actually we set this for the enrolment flow as well, nevermind

Reply:
Thanks for the review. Exactly, I changed the code to be exactly the same for enrollment. The property mappings work perfect here.